src/app/modules/estadisticas/components/rup/rup-pacientes.component.ts

```diff
@@ -174,7 +173,7 @@ export class RupPacientesComponent implements AfterViewInit, OnInit {
         // Filtramos el informa del encuentro
         this.registros.forEach(row => {
             const names = [];
-            row.relaciones.filter(e => e.concepto.conceptId !== '371531000').forEach((item) => {
+            row.relaciones.filter(e => e.concepto?.conceptId !== '371531000').forEach((item) => {
                 this.registros.forEach((reg) => {
                     const i = reg.ids.indexOf(item.id);
```

Comment:
Las relaciones son un Array de ID, pero se estaban guardando como objetos. Capaz que con un simple arreglo ya queda bien.
```suggestion
                    const idRelacion = typeof item === 'string' ? item : item.id;
                    const i = reg.ids.indexOf(idRelacion);
```

Reply:
> Las relaciones son un Array de ID, pero se estaban guardando como objetos. Capaz que con un simple arreglo ya queda bien.

Probe esto Marian pero se rompe la vista

Reply:
Entonces hay que verlo mas en detalle